examples/autoqasm/6_Customize_gate_calibrations.ipynb

```diff
@@ -0,0 +1,404 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "dda432a3",
+   "metadata": {},
+   "source": [
+    "# Customize gate calibrations\n",
+    "\n",
+    "Quantum hardware researches frequently require knowledge of native gate implementations on a specific devices, in the form of pulse sequences. The pulse implementation of gates is called \"gate calibrations\". In Amazon Braket, you have access to native gate calibrations provided by quantum hardware providers. Gaining access to gate calibrations empowers researchers to experiment with quantum gate designs and execute quantum programs with customized, improved gate calibrations."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "id": "30fd0b59",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "\n",
+    "import braket.experimental.autoqasm as aq\n",
+    "from braket.experimental.autoqasm import pulse\n",
+    "from braket.experimental.autoqasm.instructions import x, h, y, z, rx, rz, cz, measure\n",
+    "\n",
+    "from braket.aws import AwsDevice\n",
+    "from braket.devices import Devices\n",
+    "from braket.circuits import Gate, QubitSet, FreeParameter\n",
+    "from braket.pulse import DragGaussianWaveform\n",
+    "\n",
+    "\n",
+    "# Rigetti Aspen-M-3 device is the targeted device for this example notebook\n",
+    "device = AwsDevice(Devices.Rigetti.AspenM3)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f9fdfd5e",
+   "metadata": {},
+   "source": [
+    "## Gate calibrations from hardware providers"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "05c502a9",
+   "metadata": {},
+   "source": [
+    "Let's start by viewing the native gate calibrations supplied by hardware providers. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "621e631d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "provider_calibrations = device.gate_calibrations"
```

Comment:
Looks like this could be moved to the following code block, since you don't talk about this line in particular